docs/guides/knative-privatelink.md

```diff
@@ -0,0 +1,136 @@
+# Creating a PrivateLink to Knative Services
+
+[PrivateLink](https://aws.amazon.com/privatelink) is a service that enables you to establish private connectivity 
+between VPCs and services hosted on Amazon Web Services (AWS) or on-premises, 
+without exposing data to the internet. Let’s look at the configuration required
+to access Knative services deployed on an AWS Elastic Kubernetes Service (AWS) 
+cluster across AWS accounts over a PrivateLink connection.
+
+## Configuring the Istio Ingress Gateway
+
+In the default configuration, the Istio networking layer for Knative creates an 
```

Comment:
Maybe worth mention something like 

"the Istio networking layer (or other such as Contour or Kourrier)"